lib/modules/pageNavigator.js

```diff
@@ -95,9 +98,8 @@ function backToNewCommentArea() {
 	Floater.addElement($backToNewCommentArea);
 }
 
-function showLinkTitle() {
+const showLinkTitle = _.once(submissionThing => {
```

Comment:
Instead of `_.once`, how about validating that `submissionThing` is a child of `#siteTable`?

Reply:
The `_.once` construction is also used by hideChildComments to determine the submission thing, so I figure we might as well use it. While I didn't see the ad myself, the screenshot in the issue shows that the `hide all child comments` button was placed correctly so I think solution is sufficient.

Reply:
Gets the job done! If the ads team happens to add another ad unit that's sooner in the DOM (unlikely) we can revisit it.